compiler/src/dmd/backend/arm/cod4.d

```diff
@@ -1264,8 +1264,10 @@ void cdcnvt(ref CGstate cg, ref CodeBuilder cdb,elem* e, ref regm_t pretregs)
         case OPd_u64:                               // fcvtzu d31,d31 // fmov x0,d31
         L2:
             regm_t retregs1 = ALLREGS; //INSTR.FLOATREGS;
-retregs1 = mCX;  // hack because no floating support in rest of code
-//          codelem(cgstate,cdb,e.E1,retregs1,false);
+static if (1)
```

Comment:
why a bunch of these?

Reply:
Because the rest of the floating point code generation doesn't work, so I put in a bypass for it for the moment so I could test the code.